docker/planet/upgrade.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Bash is not installed on the docker image, thats why we are using sh

Reply:
Ok, will start a quick PR to install it.  `sh` doesn't support arrays, so throws an error.